isis/tests/Fixtures.cpp

```diff
@@ -1550,4 +1550,61 @@ namespace Isis {
       delete cubeList;
     }
   }
+
+  void ClipperWacFcCube::SetUp() {
+    TempTestingFiles::SetUp();
+
+    // Use defaultCube label and isd for now
+    std::ifstream isdFile("data/defaultImage/defaultCube.isd");
+    std::ifstream cubeLabel("data/defaultImage/defaultCube.pvl");
+    isdFile >> isd;
+    cubeLabel >> label;
+    
+    // Define WAC cube
+    wacFcCube = new Cube();
+    FileName wacFn(tempDir.path() + "/ClipperWAC.cub");
+    wacFcCube->fromIsd(wacFn, label, isd, "rw");
+
+    PvlGroup &wacKernels = wacFcCube->label()->findObject("IsisCube").findGroup("Kernels");
+    wacKernels.findKeyword("NaifFrameCode").setValue("-159104");
+
+      // Instruments Group
+    PvlGroup &realWacInst = wacFcCube->label()->findObject("IsisCube").findGroup("Instrument");
+    std::istringstream newWacInst(R"(
+      Group = Instrument
+        SpacecraftName            = Clipper
+        InstrumentId              = EIS-WAC-FC
+        TargetName                = Europa
+        StartTime                 = 2025-01-01T00:00:00.000
+      End_Group
+    )");
+    PvlGroup newWacInstPvl; newWacInst >> newWacInstPvl;
+    realWacInst = newWacInstPvl;
+
+      // NaifKeywords Group
+    PvlObject &realWacNk = wacFcCube->label()->findObject("NaifKeywords");
+    std::istringstream newWacNk(R"(
+      Object = NaifKeywords
+        BODY_CODE               = 502
+        BODY502_RADII           = (1562.6, 1560.3, 1559.5)
+        BODY_FRAME_CODE         = 10024
+        INS-159104_FOCAL_LENGTH = 150.40199
+        INS-159104_PIXEL_PITCH  = 0.014
+        INS-159104_TRANSX       = (0.0, 0.014004651, 0.0)
+        INS-159104_TRANSY       = (0.0, 0.0, 0.01399535)
+        INS-159104_ITRANSS      = (0.0, 71.404849, 0.0)
+        INS-159104_ITRANSL      = (0.0, 0.0, 71.4523)
+      End_Object
+    )");
+    PvlObject newWacNkPvl; newWacNk >> newWacNkPvl;
+    realWacNk = newWacNkPvl;
+
+    wacFcCube->reopen("rw");
```

Comment:
Thought we had found that reopen doesn't write the modified label to the cube before closing it. Has that behavior been modified or am I remembering wrong?

Reply:
I'm not familiar with that issue. In this case the label is identical and correctly modified wether using a reopen call or
```
    QString fileName = wacFcCube->fileName();
    delete wacFcCube;
    wacFcCube = new Cube(fileName, "rw");
```
But I've only verified this on my osx build.

If you think it'd be better I can push the above change.